pkg/build/apk.go

```diff
@@ -54,13 +54,24 @@ func (*Context) loadSystemKeyring(locations ...string) ([]string, error) {
 	}
 	for _, d := range locations {
 		keyFiles, err := os.ReadDir(d)
-		if err != nil && !errors.Is(err, os.ErrNotExist) {
+
+		if errors.Is(err, os.ErrNotExist) {
+			log.Printf("%s doesn't exist, skipping...", d)
+			continue
```

Comment:
this wasn't returning early before but since len(nil) and for/range over nil are valid in Go there wasn't a nil pointer dereference